src/main/java/org/jenkinsci/remoting/util/Timeout.java

```diff
@@ -0,0 +1,108 @@
+/*
```

Comment:
This class is an adapted version of https://github.com/jenkinsci/workflow-support-plugin/commit/c810b3874134f60be670d1205b6673fde5003c14